spec.md

```diff
@@ -1080,6 +1081,135 @@ A dictionary value has these methods:
 * [`values`](#dict·values)
 
 
+### Sets
+
+A set is a mutable, iterable collection of unique values - the set's *elements*.
+The [type](#type) of a set is `"set"`.
+
+Sets provide constant-time operations to insert, remove, or check for the
+presence of a value. Sets are implemented using a hash table, and therefore,
+just like keys of a [dictionary](#dictionaries), elements of a set must be
+hashable. A value may be used as an element of a set if and only if it may be
+used as a key of a dictionary.
+
+Sets may be constructed using the [set()](#set) built-in function.
+
+The `in` and `not in` operations check whether a value is in a set:
+
+```python
+s = set(["a", "b", "c"])
+"a" in s  # True
+"z" in s  # False
+```
+
+A set is an iterable sequence. Its length can be retrieved using the
+[len()](#len) built-in function, and the order of iteration is the order in
+which elements were added to the set:
```

Comment:
nit: "were added to the set" -> "were first added to the set"

Reply:
Fixed.